app/views/docs/getting-started-for-server.phtml

```diff
@@ -479,6 +554,25 @@ res.then((response) {
 });</code></pre>
         </div>
     </li>
+    <li>
+        <h3>Kotlin</h3>
+        <div class="ide margin-top-small" data-lang="dart" data-lang-label="Kotlin SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>import io.appwrite.Client
+import io.appwrite.services.Users
+
+val client = Client();
+val users = Users(client);
+
+client
```

Comment:
It may not matter much but the client `initialisation` should happen before `Users()` is initialised.

Reply:
It doesn't matter as long as it happens before request.

Reply:
Yes, but its a better convention to initialise an object before passing it on as a dependency to other constructors. 

Also we need to stay consistent with the Node, PHP, Ruby, Python examples. 

Looking at it now, even the deno and dart examples need to be updated to have the client initialised before passing it to storage.

It can be done in a separate PR to keep things independent